staging/src/k8s.io/api/flowcontrol/v1beta1/register.go

```diff
@@ -54,5 +55,16 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&PriorityLevelConfigurationList{},
 	)
 	metav1.AddToGroupVersion(scheme, SchemeGroupVersion)
+
+	scheme.SetGroupVersionLifecycle(SchemeGroupVersion, schema.APILifecycle{
+		IntroducedVersion: version.MajorMinor(1, 20),
+		RemovedVersion:    version.MajorMinor(1, 26),
+	})
+
+	scheme.SetResourceLifecycle(SchemeGroupVersion.WithResource("flowschema"), &FlowSchema{})
+	scheme.SetResourceLifecycle(SchemeGroupVersion.WithResource("flowschemas"), &FlowSchemaList{})
+	scheme.SetResourceLifecycle(SchemeGroupVersion.WithResource("prioritylevelconfiguration"), &PriorityLevelConfiguration{})
+	scheme.SetResourceLifecycle(SchemeGroupVersion.WithResource("prioritylevelconfigurations"), &PriorityLevelConfigurationList{})
```

Comment:
This is just an example of adding  lifecycle information about an API? How are we planning to add and maintain this for all the other APIs?  Do we need a section in the KEP about this?

Reply:
This is just an example. 
For new APIs, we should require setting lifecyle. But for apis predates compVer, I think it could be optional, we would just treat unspecified as existing forever.

Reply:
OK, that sounds good.  Strictly speaking do we need to list out all the individual resources here ("flowschema", "flowschemas", ...)?  If not would it be better to leave them absent and include a comment like `// all individual schemas of this group share the lifecycle of the group and do not require their own lifecycles to be specified.` ?